x-pack/plugins/ingest_manager/public/plugin.ts

```diff
@@ -28,10 +28,7 @@ export type IngestManagerSetup = void;
  */
 export interface IngestManagerStart {
   registerDatasource: typeof registerDatasource;
-  success: boolean;
-  error?: {
-    message: string;
-  };
+  success: Promise<true>;
```

Comment:
I found the naming a little confusing, what do you thing to have something like `initializationPromise` ?

Reply:
Thanks for the reminder. I agree that the name could be improved. Perhaps we could use the `isInitialized` or `isReady` names we use in the setup responses[1]?

I was initially trying to make smallest external change but we already made it a breaking change by making it a promise so I guess we might as well name it what we want.

[1] I should file a ticket to make these consistent